axon-server-connector/src/main/java/org/axonframework/axonserver/connector/processor/EventProcessorController.java

```diff
@@ -16,60 +17,132 @@
+        return eventProcessingConfiguration.eventProcessor(processorName)
+                                           .orElseThrow(() -> new RuntimeException(
+                                                   "Processor [" + processorName + "] not found"
+                                           ));
+    }
+
+    void pauseProcessor(String processorName) {
+        getEventProcessor(processorName).shutDown();
+        this.pauseHandlers.forEach(consumer -> consumer.accept(processorName));
+    }
+
+    void startProcessor(String processorName) {
+        getEventProcessor(processorName).start();
+        this.startHandlers.forEach(consumer -> consumer.accept(processorName));
     }
 
-    public void pauseProcessor(String processor){
-        getEventProcessor(processor).shutDown();
-        this.pauseHandlers.forEach(consumer -> consumer.accept(processor));
+    void releaseSegment(String processorName, int segmentId) {
+        EventProcessor eventProcessor = getEventProcessor(processorName);
+        if (!(eventProcessor instanceof TrackingEventProcessor)) {
+            logger.info("Release segment requested for processor [{}] which is not a Tracking Event Processor");
+            return;
+        }
+        ((TrackingEventProcessor) eventProcessor).releaseSegment(segmentId);
     }
 
-    public void startProcessor(String processor){
-        getEventProcessor(processor).start();
-        this.startHandlers.forEach(consumer -> consumer.accept(processor));
+    void splitSegment(String processorName, int segmentId) {
+        EventProcessor eventProcessor = getEventProcessor(processorName);
+        if (!(eventProcessor instanceof TrackingEventProcessor)) {
+            logger.info("Split segment requested for processor [{}] which is not a Tracking Event Processor");
+            return;
+        }
+
+        ((TrackingEventProcessor) eventProcessor).splitSegment(segmentId).whenComplete(
+                (result, throwable) -> {
+                    if (throwable != null) {
+                        logger.error("Failed to split segment [{}] for processor [{}]",
+                                     segmentId, processorName, throwable);
+                        return;
+                    }
+
+                    if (result) {
+                        logger.info("Successfully split segment [{}] of processor [{}]", segmentId, processorName);
+                    } else {
+                        logger.warn("Was not able to split segment [{}] for processor [{}]", segmentId, processorName);
+                    }
+                }
+        );
     }
 
-    public void releaseSegment(String processor, int segmentId){
-        EventProcessor eventProcessor = getEventProcessor(processor);
-        if (eventProcessor instanceof TrackingEventProcessor){
-            ((TrackingEventProcessor) eventProcessor).releaseSegment(segmentId);
+    void mergeSegment(String processorName, int segmentId) {
```

Comment:
Return the result of the merge action

Reply:
Same response as for the `splitSegment()` method from my part.